### DISCUSSION TITLE
analyst install

### DISCUSSION THREAD

**Discussion by gcsrellis** [Security-Onion-Solutions/securityonion#2214] [Unsupported Versions] [answered]
I'm a bit confused on the analyst vm install.  I try to run so-analyst-install after the .iso completes the initial install, but it doesn't work.  I thought you could do this before you install a node roll?

**Comment by TOoSmOotH** [comment, marked as answer]
It currently doesn't work in this mode. Easiest way is to do a CentOS minimal an then run the analyst install.

**Reply by pappajohndd** [reply]
So install CentOS 7 then mount the usb drive with the SO iso and run so-analyst-install from the SO folder?

**Reply by TOoSmOotH** [reply]
You can do that or clone the repo and install from there @pappajohndd

**Comment by pappajohndd** [comment]
Thank you sir
